### DISCUSSION TITLE
Plugin logs CAPBridgeViewController, UIViewController, MainActivity and UIAlertController?

### DISCUSSION THREAD

**Discussion by mariusbolik** [capawesome-team/capacitor-firebase#381] [Q&A]
Hello!
I really like this plugin. Thanks a lot for the great work!

But why does is log CAPBridgeViewController, UIViewController, MainActivity and UIAlertController?
How can I understand this?

![bildschirmfoto_2023-05-30_um_09 19 37](https://github.com/capawesome-team/capacitor-firebase/assets/39862171/f25900f1-9d35-4cc5-9125-0a83c7191d2f)

Regards,
Marius

**Comment by robingenz** [comment]
Thanks a lot!
The column shows "Seitenpfad **und** Bildschirmklasse" (Page path **and** Screen class) as described above (see screenshot).
The screen class is detected automatically and is `MainActivity` on Android or `CAPBridgeViewController`/`UIViewController` on iOS.
As far as I know Firebase currently does not offer the possibility to display only the page path.
But you can try the following:

```ts
import { FirebaseAnalytics } from '@capacitor-firebase/analytics';

const setCurrentScreen = async () => {
  await FirebaseAnalytics.setCurrentScreen({
    screenName: 'Login',
    screenClassOverride: '', // <-- Empty string
  });
};
```

This way, no screen class should be tracked anymore. I haven't tested this but maybe it works.
